### PR TITLE
Add pros and cons of the two proposals

### PR DESCRIPTION
**Pull request by aheejin**

### CODE REVIEW

File: proposals/Exceptions.md [hunk 1]
```diff
@@ -724,3 +724,30 @@ follows:
 
 The event names subsection is a `name_map` which assigns names to a subset of
 the event indices (Used for both imports and module-defined).
+
+
+---
+
+## Comparisons of the two proposals
+
+- Proposal 2 has better expressiveness and possibly provides more flexibility
```
Comment (titzer) [inline proposals/Exceptions.md]: s/has better expressiveness/is more expressive/

File: proposals/Exceptions.md [hunk 2]
```diff
@@ -724,3 +724,30 @@ follows:
...
+  manage lifetime of exception objects, from which arises several questions.
+  - How do we manage exception objects' lifetime in non-GC embeddings? Do we
+    make reference counting mandatory?
+  - Who is responsible for deleting exception objects?
+  - What should we do for except_ref values of invalid exception objects already
+    deleted?
+  - How should exception reference type be related to the existing reference
+    type or GC proposal?
+
+- In Proposal 2, the unwinder has to stop at every call stack frame with `catch`
```
Comment (titzer) [inline proposals/Exceptions.md]: s/has to/must/

File: proposals/Exceptions.md [hunk 3]
```diff
@@ -724,3 +724,30 @@ follows:
...
+  instruction because the tag matching happens within a `catch` block, whereas
+  in Proposal 1 the unwinder does not need to stop at call stack frames that
+  does not contain `catch`s with the current exception's tag. Stopping at every
```
Comment (titzer) [inline proposals/Exceptions.md]: s/does/do/

File: proposals/Exceptions.md [hunk 4]
```diff
@@ -724,3 +724,30 @@ follows:
...
+  call frame possibly degrade performance.
```
Comment (titzer) [inline proposals/Exceptions.md]: s/possibly/might/

File: proposals/Exceptions.md [hunk 5]
```diff
@@ -724,3 +724,30 @@ follows:
...
+
+- It is suggested that Proposal 2 may be more compatible with effect handlers,
+  which can be possibly added to wasm in the future.
```
Comment (titzer) [inline proposals/Exceptions.md]: s/can be possibly/might be/

**Review by titzer — APPROVED** [review]
lgtm with nits

File: proposals/Exceptions.md (hunk 5 above)
Comment (aheejin) [reply proposals/Exceptions.md]: Done

File: proposals/Exceptions.md (hunk 4 above)
Comment (aheejin) [reply proposals/Exceptions.md]: Done

File: proposals/Exceptions.md (hunk 3 above)
Comment (aheejin) [reply proposals/Exceptions.md]: Done

File: proposals/Exceptions.md (hunk 2 above)
Comment (aheejin) [reply proposals/Exceptions.md]: Done

File: proposals/Exceptions.md (hunk 1 above)
Comment (aheejin) [reply proposals/Exceptions.md]: Done

**Review by dschuff — APPROVED** [review]

**Pull request merged**